src/status_im2/contexts/quo_preview/drawers/bottom_actions.cljs

```diff
@@ -0,0 +1,57 @@
+(ns status-im2.contexts.quo-preview.drawers.bottom-actions
+  (:require [react-native.core :as rn]
+            [quo2.core :as quo]))
+
+(def button-one "Cancel")
+(def button-two "Request to join")
+(def description "Joining the community will reveal your public addresses to the node owner")
+
+(def bottom-actions-props-list
+  [{:actions          :2-actions
+    :description      description
+    :button-one-label button-one
+    :button-two-label button-two
+    :scroll           false}
+   {:actions          :1-action
+    :description      description
+    :button-two-label button-two
+    :scroll           false}
+   {:actions          :2-actions
+    :button-one-label button-one
+    :button-two-label button-two
+    :scroll           false}
+   {:actions          :1-action
+    :button-two-label button-two
+    :scroll           false}
+   {:actions          :2-actions
+    :description      description
+    :button-one-label button-one
+    :button-two-label button-two
+    :scroll           true}
+   {:actions          :1-action
+    :description      description
+    :button-two-label button-two
+    :scroll           true}
+   {:actions          :2-actions
+    :button-one-label button-one
+    :button-two-label button-two
+    :scroll           true}
+   {:actions          :1-action
+    :button-two-label button-two
+    :scroll           true}])
+
+(defn render-bottom-action-item
+  [props]
+  [quo/bottom-actions props])
+
+(defn render-fn
+  [props]
+  [quo/bottom-actions props])
+
+(defn view
```

Comment:
hmm, ideally you can use an approach similar to:
https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/quo_preview/wallet/account_origin.cljs#L27

where it's making use of the `preview/preview-container` functionality. Generally we like to have these screens working with the toggles buttons/inputs to set the configuration of the component

Reply:
Great tip!